src/pages/Settings/TwoStepVerificationPage.vue

```diff
@@ -100,7 +108,7 @@
 							<p>{{ cardSubhead }}</p>
```

Comment:
I think it would be good to have a loading placeholder for this sub-section as well, since `cardSubhead` might change after the loading completes

Reply:
<img width="821" alt="Screen Shot 2021-01-07 at 10 28 23 AM" src="https://user-images.githubusercontent.com/1521381/103930542-c1856b00-50d3-11eb-88a9-a4c323a14120.png">
3rd loading placeholder added, thanks for this suggestion.